schedule/yam/agama/agama_lvm_installation.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: agama_lvm_installation
```

Comment:
we can remove "installation" from all names, due to agama only used for installation.

Reply:
updated code and VR.